test/plugin/test_in_exec.rb

```diff
@@ -240,4 +240,22 @@ def create_driver(conf)
       assert_equal [tag, time, record], event
     }
   end
+
+  test 'emit error message with read_with_stderr' do
+    d = create_driver %[
+      tag test
+      command ruby #{File.join(File.dirname(SCRIPT_PATH), 'foo_bar_baz_no_exist.rb')}
```

Comment:
nits: `exist` is verb. no_xxx should use noun form in xxx. `existence` is noun form for `exist`.